pandas/tests/indexing/test_categorical.py

```diff
@@ -170,23 +166,6 @@ def test_slicing_and_getting_ops(self):
         res_val = df.loc["j", "cats"]
         assert res_val == exp_val
 
-        # ix
-        # frame
-        # res_df = df.loc["j":"k",[0,1]] # doesn't work?
-        res_df = df.loc["j":"k", :]
-        tm.assert_frame_equal(res_df, exp_df)
-        assert is_categorical_dtype(res_df["cats"].dtype)
-
-        # row
-        res_row = df.loc["j", :]
-        tm.assert_series_equal(res_row, exp_row)
-        assert isinstance(res_row["cats"], str)
-
-        # col
-        res_col = df.loc[:, "cats"]
-        tm.assert_series_equal(res_col, exp_col)
-        assert is_categorical_dtype(res_col.dtype)
```

Comment:
most likely these should be re-purposed as tests for loc

Reply:
We have the exact same tests a few lines above, would not have deleted them otherwise (line 149:167)

Reply:
Was duplicated code

Reply:
thanks for double-checking